fluss-common/src/main/java/com/alibaba/fluss/config/ConfigOptions.java

```diff
@@ -1309,6 +1309,24 @@ public class ConfigOptions {
                             "The column name of the version column for the `versioned` merge engine. "
                                     + "If the merge engine is set to `versioned`, the version column must be set.");
 
+    public static final ConfigOption<Boolean> LOG_TABLE_ALLOW_CREATION =
+            key("log.table.allow-creation")
```

Comment:
How about `allow.create.log.tables` and `allow.create.kv.tables`? This makes sure both of them be able to be together on docs when we support auto generate docs in the future. 

And please put the ConfigOptions after `AUTO_PARTITION_CHECK_INTERVAL`? Because `table.*` are table-level configs, not server-side configs.